src/app/core/component/candidate-renderer/candidate-renderer.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="candidate-card-container">
+    <mat-card  *ngFor="let candidate of candidates">
+        <mat-card-title>{{candidate.name}}</mat-card-title>
+        <mat-card-subtitle> Language: {{candidate.patternLanguageName}}</mat-card-subtitle>
+        <mat-card-content>
+            <p>
+                {{candidate.content['Context']}}
```

Comment:
Since the type of content (of a candidate) is json, we should provide a fallback if no context is specified